packages/graphics/vulkan-loader/package.mk

```diff
@@ -0,0 +1,40 @@
+################################################################################
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2016 Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="vulkan-loader"
+PKG_VERSION="1.2.173"
+PKG_SHA256="a64fdd5ba78ca6cd168bbcfed9b7fc24d1eac4146db4b795299d4ea92815e470"
+PKG_ARCH="any"
+PKG_LICENSE="Apache 2.0"
+PKG_SITE="https://www.khronos.org"
+PKG_URL="https://github.com/KhronosGroup/Vulkan-Loader/archive/v$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain cmake:host Python3 vulkan-headers xrandr"
+PKG_SECTION="graphics"
+PKG_SHORTDESC="Vulkan Installable Client Driver (ICD) Loader."
+
+PKG_IS_ADDON="no"
+PKG_AUTORECONF="no"
+
+PKG_CMAKE_OPTS_TARGET="-DBUILD_WSI_WAYLAND_SUPPORT=off \
+                       -DBUILD_WSI_XCB_SUPPORT=on \
+                       -DBUILD_WSI_XLIB_SUPPORT=on \
+                       -DBUILD_WSI_MIR_SUPPORT=off \
+                       -DBUILD_TESTS=off \
+                       -DBUILD_LAYERS=on \
+                       -DBUILD_DEMOS=on \
+                       -DBUILD_VKJSON=on"
```

Comment:
Add:
- `-DVulkanHeaders_INCLUDE_DIR=${SYSROOT_PREFIX}/usr/include`

Source:
`CMakeLists.txt`
```
if (TARGET Vulkan::Headers)
    message(STATUS "Using Vulkan headers from Vulkan::Headers target")
    get_target_property(VulkanHeaders_INCLUDE_DIRS Vulkan::Headers INTERFACE_INCLUDE_DIRECTORIES)
    get_target_property(VulkanRegistry_DIR Vulkan::Registry INTERFACE_INCLUDE_DIRECTORIES)
else()
    find_package(VulkanHeaders)
    if(NOT ${VulkanHeaders_FOUND})
        message(FATAL_ERROR "Could not find Vulkan headers path. This can be fixed by setting VULKAN_HEADERS_INSTALL_DIR to an "
                            "installation of the Vulkan-Headers repository.")
    endif()
    if(NOT ${VulkanRegistry_FOUND})
        message(FATAL_ERROR "Could not find Vulkan registry path. This can be fixed by setting VULKAN_HEADERS_INSTALL_DIR to an "
                            "installation of the Vulkan-Headers repository.")
    endif()
    
    # set up the Vulkan::Headers target for consistency
    add_library(vulkan-headers INTERFACE)
    target_include_directories(vulkan-headers SYSTEM INTERFACE "${VulkanHeaders_INCLUDE_DIRS}")
    add_library(Vulkan::Headers ALIAS vulkan-headers)
endif()
```

Drop:
- `-DBUILD_DEMOS`
- `-DBUILD_LAYERS`
- `-DBUILD_VKJSON`
- `-DBUILD_WSI_MIR_SUPPORT`

Source:
```
CMake Warning:                                                                                                                  
  Manually-specified variables were not used by the project:

    BUILD_DEMOS
    BUILD_LAYERS
    BUILD_VKJSON
    BUILD_WSI_MIR_SUPPORT
```

Reply:
Done